examples/oapp/README.md

```diff
@@ -73,3 +73,31 @@ Or adjust the `package.json` to for example remove `hardhat` tests:
 - "test:hardhat": "$npm_execpath hardhat test"
 + "test": "forge test"
 ```
+
+## 2) Deploying Contracts
+
+Set up deployer wallet/account:
+
+> - Rename `.env.example` -> `.env`
+> - Replace the existing 12 word mnemonic with yours, this will be used to deploy your contracts.
+> - Fund this address with the corresponding chain's native tokens you want to deploy to.
```

Comment:
This in markdown means a blockquote, I think you might just want a list (remove the `>`)

Also we now have either `MNEMONIC` or `PRIVATE_KEY`, maybe we could keep this one vague (e.g. follow the instructions in `.env.example`) or provide two examples:

#### For mnemonic-based authentication

```
MNEMONIC="bla bla bla"
```

#### For private-key-based authentication

```
PRIVATE_KEY=0x0x0xxxx
```

Reply:
updated, please check